tests/unit/conftest.py

```diff
@@ -3,9 +3,16 @@
 import pytest
 from starlette.testclient import TestClient
 
+from src.core.logging_manager import LoggingManager
 from src.main import app
 
 
+@pytest.fixture(scope="session")
+def setup(request):
```

Comment:
@vasylboliuk please create some logs and check them using pytest `caplog` fixture